activities/codebase-stewardship/criteria-for-codebase-stewardship.md

```diff
@@ -54,21 +54,21 @@ The current owners of potential codebases are asked to answer these questions to
 
 ### Community potential
 
-1. Is there a 2 to 4 year roadmap?
+1. Is there a two to four year roadmap?
 2. Is there a public backlog in English or one you can make public?
-3. How far through your current set of goals are you?
-4. How realistic vs ambitious are your goals in the next 6 months?
-5. Is there a public bugtracking system in English or one you can make public?
+3. Is there a public bugtracking system in English or one you can make public?
+4. How far through your current set of goals are you?
+5. Would you describe your goals in the next six months as realistic or ambitious?
```

Comment:
The original wording was meant to suggest a continuum - people replying in binary isn't as useful to us.